### ISSUE TITLE
Create a repo per version

### ISSUE THREAD

**Issue by jrouzierinverse** [inverse-inc/packetfence#6162]
To avoid dependencies issues when upgrading from one version to the next.
Example /etc/yum.repos.d/packetfence.repo
```
[packetfence]
name=PacketFence Repository
baseurl=http://inverse.ca/downloads/PacketFence-11.0/RHEL$releasever/$basearch
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-PACKETFENCE-CENTOS
gpgcheck=1
enabled=0
```

**Comment by julsemaan** [comment]
1000% agree with this, it will make installing older releases much more stable

**Comment by nqb** [comment]
Fixed by #6378
